Simplenote/src/main/res/layout/nav_drawer_row.xml

```diff
@@ -11,49 +11,50 @@
         android:layout_height="wrap_content"
         android:gravity="center_vertical"
         android:orientation="horizontal"
-        android:paddingBottom="@dimen/padding_small"
-        android:paddingEnd="@dimen/padding_large"
-        android:paddingLeft="@dimen/list_padding_left"
-        android:paddingRight="@dimen/padding_large"
         android:paddingStart="@dimen/list_padding_left"
+        android:paddingLeft="@dimen/list_padding_left"
         android:paddingTop="@dimen/padding_small"
+        android:paddingEnd="@dimen/padding_large"
+        android:paddingRight="@dimen/padding_large"
+        android:paddingBottom="@dimen/padding_small"
         android:visibility="gone"
         tools:visibility="visible">
 
         <com.automattic.simplenote.widgets.RobotoMediumTextView
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_weight="1"
-            android:paddingBottom="@dimen/padding_extra_small"
             android:paddingTop="@dimen/padding_extra_small"
+            android:paddingBottom="@dimen/padding_extra_small"
             android:text="@string/tags"
             android:textColor="?attr/notePreviewColor"
-            android:textSize="@dimen/nav_drawer_item_text_size"/>
+            android:textSize="@dimen/nav_drawer_item_text_size" />
 
         <com.automattic.simplenote.widgets.RobotoMediumTextView
             android:id="@+id/edit_tags"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:background="?attr/selectableItemBackgroundBorderless"
             android:clickable="true"
+            android:focusable="true"
```

Comment:
This is another instance where adding the `android:focusable="true"` attribute is good, but can revert the rest of the changes.